extensions/wikia/UserPreferencesV2/UserPreferencesV2.class.php

```diff
@@ -147,12 +147,10 @@ static public function onGetPreferences( $user, &$defaultPreferences ) {
 			$defaultPreferences['enotifminoredits']['label-message'] = 'tog-enotifminoredits-v2';
 			$defaultPreferences = self::moveToEndOfArray( $defaultPreferences, 'enotifminoredits' );
 		}
-		if ( $wgEnableDiscussions ) {
-			if ( isset( $defaultPreferences['enotifdiscussions'] ) ) {
-				$defaultPreferences['enotifdiscussions']['section'] = 'emailv2/email-me-v2';
-				$defaultPreferences['enotifdiscussions']['label-message'] = 'tog-enotifdiscussions-v2';
-				$defaultPreferences = self::moveToEndOfArray( $defaultPreferences, 'enotifdiscussions' );
-			}
+		if ( isset( $defaultPreferences['enotifdiscussions'] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "enotifdiscussions" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)